data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/http/HttpConsumerVerticleFactory.java

```diff
@@ -153,6 +153,7 @@ private static CircuitBreakerOptions createCircuitBreakerOptions(final DataPlane
 
     // TODO create a single producer per bootstrap servers.
     producerConfigs.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, resource.getBootstrapServers());
+    producerConfigs.put(ProducerConfig.INTERCEPTOR_CLASSES_CONFIG, "io.cloudevents.kafka.PartitionKeyExtensionInterceptor.class");
```

Comment:
Can you use class directly, more than the name of the class? `PartitionKeyExtensionInterceptor.class`